linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_encrypt_offload_actionsendstreamdriverauthmode/tests/default_no_pass.fail.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+bash -x setup.sh
+
+if [[ -f encrypt.conf ]]; then
+  sed -i i/\$ActionSendStreamDriverMod//g /etc/rsyslog.d/encrypt.conf
+fi
+  sed -i i/\$ActionSendStreamDriverMod//g /etc/rsyslog.conf
```

Comment:
This `sed` is definitely doing some shady stuff with the contents of `rsyslog.conf`.

The file has a bunch of 
```
...
/$ActionSendStreamDriverMod//g
/$ActionSendStreamDriverMod//g
# Remote Logging (we use TCP for reliable delivery)
/$ActionSendStreamDriverMod//g
/$ActionSendStreamDriverMod//g
/$ActionSendStreamDriverMod//g
# remote_host is: name/ip, e.g. 192.168.0.1, port optional e.g. 10514
/$ActionSendStreamDriverMod//g
/$ActionSendStreamDriverMod//g
/$ActionSendStreamDriverMod//g
...
```